R/pkg/R/functions.R

```diff
@@ -202,14 +202,18 @@ NULL
 #'          \itemize{
 #'          \item \code{from_json}: a structType object to use as the schema to use
 #'              when parsing the JSON string. Since Spark 2.3, the DDL-formatted string is
-#'              also supported for the schema.
-#'          \item \code{from_csv}: a DDL-formatted string
+#'              also supported for the schema. Since Spark 3.0, \code{schema_of_json} or
+#'              a DDL-formatted string literal can also be accepted.
```

Comment:
hmm, `the DDL-formatted string` is different to `a DDL-formatted string literal`?